README.md

```diff
@@ -0,0 +1,80 @@
+# STOCKIST WEB APP - REACT CAPSTONE PROJECT
+
+The project, a frontend web application, catalogue of financial values using [Stock API](https://financialmodelingprep.com/developer/docs/)  and collect the data from my React Application.
+
+The application features includes:
+- Listing of Most Active Stocks
+- Listing of Most Gainer's Stocks
+- Listing of Most Loser's Stocks
+- Filter between those 3, most active, gainers and losers
+- Open individual stock item
+- Search for a particular stock item, example 'AAPL'
+- Listing of the lastest stock news
+- Dynamic routes for individual stock item, example `/stocks/AAPL`
+
```

Comment:
- [ ] like the previous reviewer noted, you are required to  provide a video presentation with the following bullet points covered.
- The student recorded a video presenting the features of the project.

- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.